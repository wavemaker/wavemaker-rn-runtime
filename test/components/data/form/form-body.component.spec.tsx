import React, { ReactNode } from 'react';
import { Text, TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import WmFormBody from '@wavemaker/app-rn-runtime/components/data/form/form-body/form-body.component';

describe('Test FormBody component', () => {
    test('Check validity of sample component', () => {
      const tree = renderer.create(<WmFormBody name="test_FormBody"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });
});