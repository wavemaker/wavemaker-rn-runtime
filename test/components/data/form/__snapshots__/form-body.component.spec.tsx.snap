// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test FormBody component Check validity of sample component 1`] = `
[
  <View
    onLayout={[Function]}
    style={
      {
        "height": 0,
        "width": 0,
      }
    }
  />,
  <View
    style={
      {
        "flex": 1,
      }
    }
  />,
]
`;
